samcli/local/docker/lambda_debug_settings.py

```diff
@@ -157,7 +157,9 @@ def get_debug_settings(debug_port, debug_args_list, runtime, options):
                         ]
                     ),
                 ],
-                debug_env_vars={},
+                debug_env_vars={
+                    "NODE_PATH": "/opt/nodejs/node_modules:/opt/nodejs/node10/node_modules:/var/runtime/node_modules"
```

Comment:
Can you add a comment explaining what this is and how it works? I can't understand